src/main/kotlin/no/nav/familie/ks/sak/kjerne/forrigebehandling/EndringIVilkårsvurderingUtil.kt

```diff
@@ -20,14 +20,12 @@ object EndringIVilkårsvurderingUtil {
             Vilkår.entries.map { vilkår ->
                 val vilkårTidslinje =
                     lagEndringIVilkårsvurderingForPersonOgVilkårTidslinje(
-                        nåværendeOppfylteVilkårResultaterForPerson =
+                        nåværendeVilkårResultaterForPerson =
                             nåværendePersonResultaterForPerson
-                                .flatMap { it.vilkårResultater }
-                                .filter { it.vilkårType == vilkår && it.resultat == Resultat.OPPFYLT },
-                        forrigeOppfylteVilkårResultaterForPerson =
+                                .flatMap { it.vilkårResultater },
+                        forrigeVilkårResultaterForPerson =
                             forrigePersonResultater
-                                .flatMap { it.vilkårResultater }
-                                .filter { it.vilkårType == vilkår && it.resultat == Resultat.OPPFYLT },
```

Comment:
Her trenger man egentlig bare droppe `it.vilkårType == vilkår`, man trenger ikke de andre endringene